### DISCUSSION TITLE
Unable to fix error

### DISCUSSION THREAD

**Discussion by M-Ipekci** [fwouts/previewjs#1524] [General]
SyntaxError: The requested module '/preview/src/index.js?t=1680311977081' does not provide an export named 'default'
This error keeps popping up and no matter what I do I can never figure out how to solve it and I don't understand what it is asking for so I need help. If you have any ideas please let me know.

**Comment by fwouts** [comment]
Hey @M-Ipekci sorry to hear you're running into trouble!

Would you be able to share some details about your set-up? What framework do you use? What does src/index.js look like?

The ideal would be if you had a repo you could share.
